app/mailers/pay/user_mailer.rb

```diff
@@ -20,6 +20,14 @@ def payment_action_required
       mail to: to
     end
 
+    def subscription_trial_will_end
+      mail to: to
+    end
+
+    def subscription_trial_ended
+      mail to: to
```

Comment:
If you get a chance, merge master into this branch and swap this to use the new `mail_arguments`